portal/sign/assets/signer_modal.php

```diff
@@ -16,11 +16,13 @@
 $is_portal = isset($_SESSION['portal_init']) ? 1 : $_GET['isPortal'];
 if (empty($is_portal)) {
     session_destroy();
-    require_once(__DIR__ . '/../../../interface/globals.php');
 } else {
-    require_once __DIR__ . "/../../verify_session.php";
+    $pid = $_SESSION['pid'];
```

Comment:
if do this, then need to ensure patient is authenticated (with current code I could pass a get parameter isPortal and then get access to this script.
recommend putting the following code in this else block:
```
//landing page definition -- where to go if something goes wrong
$landingpage = "index.php?site=" . urlencode($_SESSION['site_id'] ?? null);
//
if (isset($_SESSION['pid']) && isset($_SESSION['patient_portal_onsite_two'])) {
    $pid = $_SESSION['pid'];
} else {
    OpenEMR\Common\Session\SessionUtil::portalSessionCookieDestroy();
    header('Location: ' . $landingpage . '&w');
    exit;
}
$ignoreAuth_onsite_portal = true;
```

Reply:
oh ok, whoops but it seems like we're recreating a lot of `verify_session.php` here

Reply:
yep, but that's fine. we do this in several other scripts (especially the ones that need to work on both the patient portal and in core openemr). and it will then get rid of the notice :)